### PR TITLE
Issue625 setup py should not import fipy

### LINKED ISSUE #625: setup.py should not import fipy
[setup.py](https://github.com/usnistgov/fipy/blob/develop/setup.py#L4) imports several commands from the `fipy` package (`Efficiency_test`, `Copy_script`, `changelog`, `_TestClass`). As a result, simple commands like `python setup.py egg_info` don't work without at least some of FiPy's prerequisites, e.g., `numpy`, which is annoying, fragile, and potentially circular.

These commands should be moved either into `setup.py` or into an installation package that isn't `fipy`.

### PR DESCRIPTION
**Pull request by guyer**
Fixes #625

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,161 +3,23 @@
 from distutils.core import Command
 import versioneer
```
Comment (wd15) [inline setup.py]: This is importing the local versioneer.py version so versioneer isn't a dependency for installation.

**Comment by guyer** [comment]
> There is no `install_requires` section in the `setup.py`. Should that be there? I think it's something that we've never had, right?

I intend to address `install_requires` with #565

**Comment by guyer** [comment]
> Shouldn't the `getVersion` function in `setup.py` be dispensed with?

Gah! I made a complete hash when I rebased. I'll clean up and resubmit for review.

**Comment by guyer** [comment]
> Recommend running pylint, flake8 and black...

Good thought. I'll let you know if I need help.

**Comment by guyer** [comment]
Closing in favor of #637

**Pull request closed without merging**